### PR TITLE
Update applications with PATCH

### PR DESCRIPTION
**Pull request by harry-kim**
Refs #SDE-4562

TODO:
- [x] Set milestone label in pull request (SDE version)
- [x] Updated changelog.md
- [x] Completed documentation for the new endpoint.
- [x] Feature has been merged into SDE develop.

### CODE REVIEW

**Comment by harry-kim** [comment]
do i need to update changelog for this?
not sure the milestone..

**Comment by RouganStriker** [comment]
I think we should still update the changelog, maybe just say fixed documentation or something

+1

File: source/includes/changelog.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+
+## v4.9
+- Change the update application example to use PATCH instead of PUT
```
Comment (hanifvirani) [inline source/includes/changelog.md]: Don't think we should mention it this way. We might want to change all PUT to PATCH throughout the docs or vice versa and add a blurb about it at the top. Then mention the entire change as one in the changelog. But that can be done separately.

Comment (harry-kim) [reply source/includes/changelog.md]: so should i just remove the changelog stuff?

Comment (funkaoshi) [reply source/includes/changelog.md]: Yeah, we should probably just be using PATCH throughout our docs. And say what PUT does versus PATCH.

**Comment by emintham** [comment]
+1

File: source/includes/changelog.md (hunk 1 above)
Comment (harry-kim) [reply source/includes/changelog.md]: removed

**Pull request merged**